### ISSUE TITLE
Line Spacing

### ISSUE THREAD

**Issue by maryfries** [bjc-edc-2017-18/bjc-r#291]
At some point, we must have changed the amount of space between lines, and it's a little too tight now. Can we please revisit?

**Comment by cycomachead** [comment]
It should be ~1.5 everywhere, but it doesn't look this this is happening, in some places it seems to be more like 1.4
